### PR TITLE
StatisticsBackend::get_type [11666]

### PR DESCRIPTION
**Pull request by SerranoePros**
- StatisticsBackend::database_ changed to static.
- Added /TestUtils/ for test with similar database setup.
- After merge change load_database to use /TestUtils/.

### CODE REVIEW

File: test/unittest/StatisticsBackend/StatisticsBackendTests.cpp [hunk 1]
```diff
@@ -113,6 +230,7 @@ TEST(statistics_backend_tests, get_data_supported_entity_kinds)
     }
 }
 
+
```
Comment (JLBuenoLopez-eProsima) [inline test/unittest/StatisticsBackend/StatisticsBackendTests.cpp:233]: Remove extra blank line

**Review by JLBuenoLopez-eProsima — CHANGES_REQUESTED** [review]
After merging #61, a PR can be issued so the `load_database_tests` also use the `DatabaseUtils` auxiliary class.

**Comment by JLBuenoLopez-eProsima** [comment]
Please, complete also the [documentation for this API](https://fast-dds-statistics-backend.readthedocs.io/en/latest/rst/statistics_backend/get_type.html), removing the `Warning` notice and completing any other information you deem necessary.

File: test/unittest/StatisticsBackend/StatisticsBackendTests.cpp [hunk 2]
```diff
@@ -16,13 +16,130 @@
 
 #include <StatisticsBackend.hpp>
 #include <types/types.hpp>
+#include "database/database.hpp"
 
 #include "gtest/gtest.h"
 
+#include <DatabaseUtils.hpp>
```
[the remark this answers is not in the document]
Comment (SerranoePros) [reply test/unittest/StatisticsBackend/StatisticsBackendTests.cpp:23]: Check when push

**Comment by SerranoePros** [comment]
> Please, complete also the [documentation for this API](https://fast-dds-statistics-backend.readthedocs.io/en/latest/rst/statistics_backend/get_type.html), removing the `Warning` notice and completing any other information you deem necessary.

Check when push, please.

File: test/unittest/StatisticsBackend/StatisticsBackendTests.cpp (hunk 2 above)
[the remark this answers is not in the document]
Comment (JLBuenoLopez-eProsima) [reply test/unittest/StatisticsBackend/StatisticsBackendTests.cpp]: I should have clarified that the public headers should be before the private ones. And we usually add a blank line between them to distinguish.

File: test/unittest/StatisticsBackend/StatisticsBackendTests.cpp [hunk 3]
```diff
@@ -14,15 +14,131 @@
 
 #include <list>
 
+#include <gtest/gtest.h>
+
+#include <database/database.hpp>
+#include <DatabaseUtils.hpp>
 #include <StatisticsBackend.hpp>
 #include <types/types.hpp>
```
Comment (JLBuenoLopez-eProsima) [inline test/unittest/StatisticsBackend/StatisticsBackendTests.cpp:22]:
```suggestion
#include <StatisticsBackend.hpp>
#include <types/types.hpp>

#include <database/database.hpp>
#include <DatabaseUtils.hpp>
```

**Review by JLBuenoLopez-eProsima — COMMENTED** [review]
Only a small nitpick

File: docs/rst/statistics_backend/get_type.rst [hunk 4]
```diff
@@ -5,11 +5,7 @@
 Get entity type
 ---------------
 
-.. warning::
-    This feature is currently not supported.
-    It will be implemented on a future release of *Fast DDS Statistics Backend*.
-
-Is is also possible to retrieve the |EntityKind-api| of an entity given its |EntityId-api|:
+Is also possible to retrieve the |EntityKind-api| of an entity given its |EntityId-api|:
```
Comment (JLBuenoLopez-eProsima) [inline docs/rst/statistics_backend/get_type.rst:8]:
```suggestion
It is also possible to retrieve the |EntityKind-api| of an entity given its |EntityId-api|:
```

**Review by JLBuenoLopez-eProsima — COMMENTED** [review]
Sorry, I overlooked this small detail.

**Review by JLBuenoLopez-eProsima — APPROVED** [review]
LGTM with green CI

**Pull request merged**